populate/src/main/resources/seed/i18n/dev/languageTexts.json

```diff
@@ -67,6 +67,14 @@
   {"keyName": "suggestBetterAddressHeader", "text": "DEV_Is this the correct address?", "language": "dev"},
   {"keyName": "suggestBetterAddressBody", "text": "DEV_We were able to verify a similar address. Please select which you would like to use.", "language": "dev"},
   {"keyName": "newAddress", "text": "DEV_New Address", "language": "dev"},
-  {"keyName": "oldAddress", "text": "DEV_Old Address", "language": "dev"}
+  {"keyName": "oldAddress", "text": "DEV_Old Address", "language": "dev"},
+  {"keyName": "you", "text": "DEV_You", "language": "dev"},
+  {"keyName": "yourDependent", "text": "DEV_Your Dependent", "language": "dev"},
+  {"keyName": "youInParens", "text": "DEV_(you)", "language": "dev"},
```

Comment:
As of https://github.com/broadinstitute/juniper/pull/865, the i18n helper supports substitutions. I wish I had added it sooner, but it could make some of these language texts a little more comprehensible and easier to deal with in certain languages, so we don't have to break these down to individual words. I could see this instead being something like `DEV_${participantGivenName} ${participantFamilyName} (you)`

Totally up to you though- I'm curious what you think is more readable.

Reply:
Hmm... It would definitely be possible, but I think the logic is slightly more complex than a simple replace, so I might leave it the way it is. But this is very good to know in the future.